suites/guppy/nestedQueryTest.js

```diff
@@ -0,0 +1,242 @@
+/*
+ * Prerequisites:
+ * 1. Data in the commons are ETLed
+ * 2. Data has a nested structure. Take qa-mickey as example:
+ * "patient" doc has nested prop "ActionableMutations"
+ * 3. Guppy is up, take qa-mickey as example:
+          guppy version entry: https://github.com/uc-cdis/gitops-qa/blob/master/qa-mickey.planx-pla.net/manifest.json#L23
+          config: https://github.com/uc-cdis/gitops-qa/blob/master/qa-mickey.planx-pla.net/manifest.json#L96-L104
+          related portal config: https://github.com/uc-cdis/gitops-qa/blob/master/qa-mickey.planx-pla.net/portal/gitops.json#L223-L228
+ * 4. User got access to the website and the data, and could visit `/query` page
+ */
+Feature('Support query for nested data - https://github.com/uc-cdis/guppy/pull/52');
+
+const chai = require('chai');
+const { interactive, ifInteractive } = require('../../utils/interactive.js');
+
+const { expect } = chai;
+
+Scenario('Check guppy is up and query page is ready', ifInteractive(
+  async () => {
+    const result = await interactive(`
+        1. Goto target commons (e.g., https://qa-mickey.planx-pla.net/).
+        2. Log in, and then click "Query" in nav bar.
+        3. You should be able to see GraphiQL editor, the result panel in the right side should not have any error messages. 
+    `);
+    expect(result.didPass, result.details).to.be.true;
+  },
+));
+
+Scenario('could query nested data', ifInteractive(
```

Comment:
the executable test scenarios must contain the CodeceptJS tag " @manual "
So they won't run as part of the fully automated PR checks (integration tests).|

More details in this blog post:
https://ctds-planx.atlassian.net/wiki/spaces/~696307876/blog/2019/10/24/143949839/How+we+test+stuff#Including-and-Excluding-tests